.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
@anikdhabal Could you please revert this change if it is not needed?

Reply:
Yes, sure